src/cls/IPM/ResourceProcessor/WebApplication.cls

```diff
@@ -0,0 +1,263 @@
+Include (%sySecurity, %occErrors, %occReference)
+
+Class %IPM.ResourceProcessor.WebApplication Extends (%IPM.ResourceProcessor.Abstract, %XML.Adaptor) [ PropertyClass = %IPM.ResourceProcessor.PropertyParameters ]
+{
+
+/// Description of resource processor class (shown in UI)
+Parameter DESCRIPTION As STRING = "Facilitates Web Application create/update/delete.";
+
+/// Name of the web application.
+Property Name As %String [ Calculated ];
+
+Method NameGet() As %String
+{
+    Set tName = $Get(..SecurityProperties("Name"), $Get(..SecurityProperties("Url"), ""))
```

Comment:
Nit: for net new code, no need to use the legacy `t` prefix

Reply:
Done.